tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -494,3 +559,16 @@ Then('the file {string} should be in {string} state on the webUI after a page re
 Then('the autocomplete list should not be displayed on the webUI', function () {
   return client.page.FilesPageElement.sharingDialog().assertAutocompleteListIsNotVisible()
 })
+
+Then('only {string} should be listed in the autocomplete list on the webUI', function (username) {
+  return client.page.FilesPageElement.sharingDialog().getShareAutocompleteItemsList()
+    .then(itemsList => {
```

Comment:
Why not `assert` that there is only 1 item and the first item is the expected item? I'd think `only` as having only one item.